packages/adapter-node/src/LoadNodeFont.ts

```diff
@@ -0,0 +1,30 @@
+import { registerFont } from 'canvas';
+import { getFontFamilyName } from './temp/misc';
+
+const validFonts = ['woff', 'woff2', 'ttf', 'otf'];
+
+/**
+ * loads our textures!
+ * this makes use of imageBitmaps where available.
+ * We load the ImageBitmap on a different thread using CentralDispatch
+ * We can then use the ImageBitmap as a source for a Pixi Texture
+ */
+export const loadNodeFont = {
```

Comment:
Could you change the casing of the filename to match the export? LoadNodeFont vs loadNodeFont.

Reply:
And same for LoadNodeTexture.ts